armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1414,9 +1480,14 @@ def _summarizeGroups(self, blockCollectionsByXsGroup):
                 xsIDGroup = self._getXsIDGroup(xsID)
                 if xsIDGroup == self._REPR_GROUP:
                     reprBlock = self.representativeBlocks.get(xsID)
+                    temp = self.avgNucTemperatures.get(self.cs["xsTempIsotope"], "N/A")
                     runLog.extra(
-                        "XS ID {} contains {:4d} blocks, represented by: {:65s}".format(
-                            xsID, len(blocks), reprBlock
+                        "XS ID {} contains {:4d} blocks, with burnup {} and avg fuel temp {}, represented by: {:65s}".format(
```

Comment:
```suggestion
                        "XS ID {} contains {:4d} blocks, with avg burnup {} and avg fuel temp {}, represented by: {:65s}".format(
```

Reply:
Maybe also add units to avg burnup and avg fuel temp

Reply:
txt fixed. ill work on test

Reply:
resolved